ibis/backends/mysql/tests/test_client.py

```diff
@@ -20,8 +20,8 @@
     ("boolean", dt.int8),
     ("smallint", dt.int16),
     ("int2", dt.int16),
-    ("mediumint", dt.int32),
-    ("int3", dt.int32),
+    # ("mediumint", dt.int32), => https://github.com/tobymao/sqlglot/issues/2109
+    # ("int3", dt.int32), => https://github.com/tobymao/sqlglot/issues/2109
```

Comment:
Neither of these is fixed by 2109. What's the relationship?

Reply:
Oh, nevermind, it's just `int3` that still doesn't work.

Reply:
Yes